data.sql

```diff
@@ -1,5 +1,6 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO vet_clinic(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Agumon', '2020-2-3', 0, TRUE, 10.23);
+INSERT INTO vet_clinic(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Gabumon', '2018-11-15', 2, TRUE, 8);
+INSERT INTO vet_clinic(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Pikachu', '2021-6-12', 1, FALSE, 15.04);
+INSERT INTO vet_clinic(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Devimon', '2017-5-12', 5, TRUE, 11);
```

Comment:
- [ ] This data insertion suppose to be into the `animals` table, not vet_clinic as you did, So please fix that.
[Hint] For example:
```sql
INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Agumon', '2020-2-3', 0, TRUE, 10.23);
```